src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/IntervalArgumentCollection.java

```diff
@@ -55,7 +55,8 @@
      * (e.g. -XL myFile.intervals).
      * @return strings gathered from the command line -XL argument to be parsed into intervals to exclude
      */
-    @Argument(fullName = EXCLUDE_INTERVALS_LONG_NAME, shortName = "XL", doc = "One or more genomic intervals to exclude from processing", optional = true, common = true)
+    @Argument(fullName = EXCLUDE_INTERVALS_LONG_NAME, shortName = "XL", doc = "One or more genomic intervals to exclude from processing",
+            suppressFileExpansion = true, optional = true, common = true)
```

Comment:
Add comment here explaining why file expansion is suppressed for this arg. The comment should be a non-javadoc comment (eg., `//`) so that it won't be picked up by the doc generation code.

Reply:
done.